config/config.rb

```diff
@@ -66,6 +66,9 @@
 
   configatron.labwhere_api = 'localhost:3200/api'
 
+  configatron.baracoda_api = 'http://localhost:8000'
+  configatron.plate_barcode_prefix = 'SQPD'
```

Comment:
what's SQPD stand for, if anything? curious. Sequencescape something.
I get it's the new prefix so we can't clash with DN. But why 4 characters not 2 and therefore takes more space on labware labels?

Reply:
It's stands for Sequencescape Development. 
It can be any size, it could be SD, or anything else if we want. We thought SQPD but any other prefix is ok. But the new thing is that it uses - to split the barcode in parts.
The new format is: ```<PREFIX>-<NUMBER>``` or ```<PREFIX>-<NUMBER>-<NUMBER>```.